stochastic/continuous/brownian_motion.py

```diff
@@ -95,14 +95,19 @@ def sample(self, n, zero=True):
         :param bool zero: if True, include :math:`t=0`
         """
         return self._sample_brownian_motion(n, zero)
-
+    
     def _sample_brownian_motion_at(self, times):
         """Generate a Brownian motion at specified times."""
-        s = np.cumsum(self._sample_gaussian_noise_at(times))
+        
+        bm = np.cumsum(self.scale * self._sample_gaussian_noise_at(times))
+        
+        if self.drift != 0:
+            bm += [self.drift * t for t in times]
```

Comment:
this might be more performant if we coerce times to be a numpy array and then just multiply by the drift parameter

Reply:
Oke I will change this. I am also thinking about moving the "cumsum" to the end of the function, such that we first calculate the incrementes. In that case we can also make a if-statement for scale != 1.